docs/libcurl/opts/CURLINFO_EARLYDATA_SENT_T.md

```diff
@@ -0,0 +1,73 @@
+---
+c: Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+SPDX-License-Identifier: curl
+Title: CURLINFO_EARLYDATA_SENT_T
+Section: 3
+Source: libcurl
+See-also:
+  - curl_easy_getinfo (3)
+  - curl_easy_setopt (3)
+Protocol:
+  - All
```

Comment:
Since early data is only for TLS protocols and the feature only works for GnuTLS for now, I propose:

```suggestion
Protocol:
  - TLS
TLS-backend:
  - GnuTLS
```

Reply:
ack